evmjit/libevmjit-cpp/Env.cpp

```diff
@@ -46,23 +46,22 @@ extern "C"
 		*o_hash = _env->blockhash(llvm2eth(*_number));
 	}
 
-	EXPORT void env_create(ExtVMFace* _env, i256* io_gas, i256* _endowment, byte* _initBeg, uint64_t _initSize, h256* o_address)
+	EXPORT void env_create(ExtVMFace* _env, int64_t* io_gas, i256* _endowment, byte* _initBeg, uint64_t _initSize, h256* o_address)
```

Comment:
Not in JIT. JIT uses int64_t for gas calculations. You cannot request an execution with more than 2^63-1 gas units, but as a miner would you risk that?